src/ImageBinarization.jl

```diff
@@ -47,17 +41,11 @@ export
 
     # Algorithms
     AdaptiveThreshold, recommend_size,
-    Balanced,
-    Entropy,
-    Intermodes,
-    MinimumError,
-    MinimumIntermodes,
-    Moments,
     Niblack,
-    Otsu,
     Polysegment,
     Sauvola,
-    UnimodalRosin,
-    Yen
+    
+    # also reexport algorithms in HistogramThresholding
+    SingleHistogramThreshold
```

Comment:
The only reason I export this is because I want to make `? SingleHistogramThreshold` work.

Since I plan to introduce this package into Images.jl, we might be conservative and not export this?

Reply:
I still choose to export this for completeness.